.github/workflows/commit-msg-check.yml

```diff
@@ -31,10 +31,13 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
 
 jobs:
-  title:
-    if: github.repository == 'oneapi-src/oneDNN'
+  commit-message-check:
+    name: Commit message checker
```

Comment:
The check for oneapi-src/oneDNN was to prevent failure in private repos where the checkout action is not available.
Any reason to take it out now?

Reply:
It prevents the action from working on any forks, which may not be desirable, and hardcodes the name of the repository.  None of the actions we have make checks like this. For private repos (I guess one very specific private repo) we have an option to disable this action, as we do for others.

Reply:
Ok I see, I did not know about other ways mentioned. THanks.